src/hooks/usePotd.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'preact/hooks'
+
+export const usePotd = () => {
```

Comment:
Does "potd" stand for photo of the day?

Reply:
Pictures

Reply:
I'm all for full name variables, we don't pay per character written